### ISSUE TITLE
Blokada can interfere with Spotify

### ISSUE THREAD

**Issue by kar** [blokadaorg/blokada#175]
App worked as normal with blokada off. I had to whitelist the following to get the spotify-app work as normal:
   * b.scorecardreseach.com
   * spclient.wg.spotify.com
   * api.mixpanel.com
(trigged by turning on notifications and then open spotify and enter discovery- and radio-tabs).

**Comment by animeking9901** [comment]
This info might help modded Spotify users,good research

**Comment by iamrohanbro** [comment]
I did face the same issue a week ago, the home screen under Spotify wasn't connecting so I had to whitelist Spotify.

**Comment by d4rken** [comment]
Are you sure all 3 domains are needed? I had similar issues (homescreen/radios not loading) and it was only caused by `spclient.wg.spotify.com`.

**Comment by kar** [comment]
Indeed at least the mixpanel entry is something I wouldn't like to whitelist. This was reported by one of our users, I haven't checked it personally yet. thanks for input

**Comment by Zackptg5** [comment]
@kar 
The spclient is the only one needed. See this issue on unified hosts repo: https://github.com/StevenBlack/hosts/issues/552

**Comment by rpeter85** [comment]
Is this ticket still valid? I use 3.7 and experienced 0 issues with Spotify.

**Comment by animeking9901** [comment]
Just tested with the 3.7 on the beta channel(I normally use Amazon Prime music so I downloaded Spotify specifically to validate) It attempted to load a video ad,failed,and granted 30 min of ad free. This issue can be closed I'd say

**Comment by rpeter85** [comment]
4.1.6, still no issues. I think it cab be closed.

**Comment by 5fire5** [comment]
Is it possible that this has become an issue again?
I have been using blokada for roughly 4 months and just in the last week all spotify downloads have been failing and then app refused to connect entirely.  Disabling Blokada solved the spotify problem.  This is on 4.5.2; I have the notifications enabled and can see that it is blocking spclient.wg.spotify.com

**Comment by peterroth** [comment]
Which host lists do you use? Blokada blocks only the domains that are on the enabled lists.

**Comment by 5fire5** [comment]
Thanks for the reply.  I have tweaked my settings and come up with a solution.  To answer your question;  I was using Energized Blu for my host list, but switched to StevenBlack Unified (shout-out Kingston ON).
At the same time I switched to AdGuard DNS from Keweon.  Spotify is working now and I no longer see spclient.wg.spotify.com being blocked.  I still have one app that requires disabling Blokada in order to run correctly, namely Teamsnap.
